dynwalls/args.py

```diff
@@ -26,7 +26,8 @@ def __init__(self):
 
         self.cli = argparse.ArgumentParser(description=PROGRAM_DESC, usage=usage)
         self.cli.add_argument("action", choices=Args.actions, help="The action to perform")
-
```

Comment:
PyCharm indicated that `action` and `subparser` were initialized outside of `__init__` on line 45